lecture02.tex

```diff
@@ -25,6 +25,14 @@ \section{Lecture 2}
 Recall that the algebra $B$ is said to be \emph{of finite type} provided that $B$ is finitely generated, meaning that $B = A[b_{1},\ldots,b_{n}]$ for some choice of $b_{1},\ldots,b_{n} \in B$.
 A stronger notion of finiteness is being a \emph{finite} algebra, which means that $B$ is finitely generates as an $A$-module.
 
+\begin{lemma}[Transitivity of Finiteness]\label{lemma:transitivity of finiteness}
+	Let $A$ be a finitely generated $B$-module and $B$ a finitely generated $C$-module. Then $A$ is finitely generated as a $C$-module. 
+\end{lemma}
+\begin{proof}
+	Suppose $\{a_i\colon i= 1,\cdots, m\}$ is a generator of $A$ as a $B$-module and $\{b_i\colon i= 1,\cdots, n\}$ is a generator of $B$ as a $C$-module. Then for any $b\in B$, there exist $c_i(b)\in C$ for $i = 1,\cdots, n$ such that $b = \sum_i c_i(b)b_i$. Next, for an arbitrary $a\in A$, there exist $b_j'\in B$ for $j = 1, \cdots, m$ such that \[a = \sum_{j = 1}^m b_j' a_j = \sum_{j = 1}^m \left(\sum_{i=1}^n c_i(b_j')\cdot b_i\right)\cdot a_j = \sum_{j = 1}^m \sum_{i=1}^n c_i(b_j')\cdot b_ia_j,\]
```

Comment:
Replace `generator` by `system of generators`. 
Also, please modify the proof so that the notation matches the modified statement of the lemma.